src/styles/blocks/showcase.scss

```diff
@@ -0,0 +1,20 @@
+.showcase {
+  display: flex;
+  width: 4 * $card-width + 3 * $card-margin-right;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
You can't set margins to BEM block

Reply:
Same here, you didn't fix comment

Reply:
I dont have idea how to centre catalog without margins @Anna-Tkachenko

Reply:
You can set margins, just you need to make the catalog as an element of some parent BEM block, and set margins to the class name of an element
You can't set external styles to BEM blocks